addons/point_of_sale/tools/posbox/overwrite_before_init/etc/logrotate.conf

```diff
@@ -14,19 +14,4 @@ create
 # packages drop log rotation information into this directory
 include /etc/logrotate.d
 
-# no packages own wtmp, or btmp -- we'll rotate them here
-/var/log/wtmp {
-    missingok
-    monthly
-    create 0664 root utmp
-    rotate 1
-}
-
-/var/log/btmp {
-    missingok
-    monthly
-    create 0660 root utmp
-    rotate 1
-}
```

Comment:
These lines already exist in `/etc/logrotate.d/wtmp` and `/etc/logrotate.d/btmp` respectively, duplicating them here causes an error.